app/policies/calendar_policy.rb

```diff
@@ -42,16 +39,32 @@ def resolve
       return scope.all if user.root?
       return scope.none if !user.partner_admin? && !user.neighbourhood_admin?
```

Comment:
don't you need to add in a `!user.partnership_admin?` check here too?

Reply:
yes I think this is a good catch, all the conditionals are getting a bit hairy round here